sources/metrics/tracking pixels/TrackingPixelsGenerator.swift.erb

```diff
@@ -5,8 +5,13 @@ import Foundation <%
 require 'YAML'
 require 'FileUtils'
 
-path = "mobile sdk evolution/definitions/tracking pixels/tracking-pixels.yaml"
-api = YAML.load(File.read(path)).map { |request_name, fields|
+file = nil
+Dir.chdir('..') do 
+  path = "mobile-sdk-evolution/definitions/tracking pixels/tracking-pixels.yaml"
```

Comment:
Is it possible to avoid the hardcoded path to `mobile-sdk-evolution` as far as it is not a part of this SDK anymore?

Reply:
Maybe - it is temporary solution.